README.md

```diff
@@ -210,7 +210,7 @@ FAQ
     OpenShift is designed to run any existing Docker images.  In addition you can define builds that will produce new Docker images from a Dockerfile.  However the real magic of OpenShift can be seen when using [Source-To-Image](https://github.com/openshift/source-to-image)(STI) builds which allow you to simply supply an application source repository which will be combined with an existing STI-enabled Docker image to produce a new runnable image that runs your application.  We are continuing to grow the ecosystem of STI-enabled images and documenting them [here](https://ci.openshift.redhat.com/openshift-docs-master-testing/latest/openshift_sti_images/overview.html).  We also have a few more experimental images available:
 
     * [Wildfly](https://github.com/openshift/wildfly-8-centos)
-    * [Ruby](https://github.com/openshift/ruby-20-centos/)
+    * [Ruby](https://github.com/openshift/sti-ruby/)
```

Comment:
@bparees I've already send PR, which updates the the openshift-docs. You suggest to remove the ruby link ?